qtpy/tests/test_main.py

```diff
@@ -16,6 +20,7 @@ def assert_pyside2():
     assert QtGui.QPainter is PySide2.QtGui.QPainter
     assert QtWidgets.QWidget is PySide2.QtWidgets.QWidget
     assert QtWebEngineWidgets.QWebEnginePage is PySide2.QtWebEngineWidgets.QWebEnginePage
+    assert os.environ['QT_API'] == 'PySide2'
```

Comment:
I think the tests are failing due to lower vs uppercase names for the bindings
```suggestion
    assert os.environ['QT_API'] == 'pyside2'
```